managers/oclcCatalog.py

```diff
@@ -27,6 +35,47 @@ def queryCatalog(self):
                 classifyResp.status_code
             ))
             return None
-    
+
         return classifyResp.text
-        
\ No newline at end of file
+
+    def getToken(self):
+        try:
+            authResp = requests.post(
+                self.OCLC_AUTH_URL,
+                headers={ 'Content-Type': 'application/x-www-form-urlencoded'},
+                auth=(self.oclcClientID, self.oclcClientSecret),
+            )
+        except (Timeout, ConnectionError):
+            print(f'Failed to retrieve token from {self.OCLC_AUTH_URL}')
+            return None
+
+        if authResp.status_code != 200:
+            print('OCLC token retrieval failed with status {}'.format(
+                authResp.status_code
+            ))
+            return None
+
+        self.token = authResp.json()['access_token']
+        self.tokenExpiresAt = authResp.json()['expires_at']
+
+    def queryBriefBibs(self, query):
+        """Accepts a query in the form of an OCLC keyword search or fielded search"""
+        self.getToken()
+        bibsEndpoint = self.OCLC_SEARCH_URL + 'brief-bibs'
+
+        headers = {"Authorization": f"Bearer {self.token}"}
+        try:
+            bibsResp = requests.get(
+                bibsEndpoint,
+                headers=headers,
+                params={'q': query}
+            )
+        except (Timeout, ConnectionError):
+            print(f'Failed to query {bibsEndpoint} with query {query}')
+            return self.queryCatalog()
+
+        if bibsResp.status_code != 200:
+            print(f'OCLC Catalog Request failed with status {bibsResp.status_code}')
+            return None
+
+        return bibsResp.json()
```

Comment:
I went ahead and returned the dict from json here. This is asymmetric with the other function returning data in the file, but that function hands over to an XML parser, whereas we just want the dict values.